server/routes/s3-routes.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+const s3_controller = require('../controllers/s3-controller');
+const checkAuth = require('../middleware/check-auth');
+
+// router.use(checkAuth);
```

Comment:
will add this back on as soon as the button gets integrated to the receipt creation page